### ISSUE TITLE
Build a reference for the cellranger.wdl

### ISSUE THREAD

**Issue by samanehsan** [HumanCellAtlas/secondary-analysis#275]
The references available for download from the cellranger website are not openly licensed, so we will need to build a custom GRCh38 reference from https://www.gencodegenes.org/releases/current.html using the cellranger mkref command: https://support.10xgenomics.com/single-cell-gene-expression/software/pipelines/latest/advanced/references.

**Comment by rexwangcc** [comment]
The AC for this ticket is getting ambiguous to me, the PR is [there](https://github.com/HumanCellAtlas/skylab/pull/130) and hopefully will get merged in soon. However, I know there are some ongoing conversations between @jishuxu @mckinsel and @ambrosejcarr about how we can dig deeper into the cellranger tool. So when we should we close this ticket then? @kbergin I feel like this is one of the tickets that requires an uncertain amount of research effort.

**Comment by kbergin** [comment]
Yes, when written it seemed clear, but with the new information we have it is a bit more ambiguous. I have made a separate ticket for investigating getting gencode as our reference #338 . I think it is worthwhile to merge in the ensembl reference code to enable us to start doing testing in the Green Box with the 10x wdl. It will just not be able to be merged into prod, possibly even staging, open to opinions on the latter.

Does that make sense?
